src/components/stepforms/schemas/evolution.ts

```diff
@@ -0,0 +1,70 @@
+import { addNotInColumnNamesConstraint, StepFormType } from './utils';
+
+const createSchema = () => ({
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  title: 'Evolution step',
+  type: 'object',
+  properties: {
+    name: {
+      type: 'string',
+      consnt: 'evolution',
+    },
+    dateCol: {
+      type: 'string',
+      minLength: 1,
+      title: 'Date column',
+      description: 'Date column used to compute evolution',
+      attrs: {
+        placeholder: 'Select a column',
+      },
+    },
+    valueCol: {
+      type: 'string',
+      minLength: 1,
+      title: 'Value column',
+      description: 'The value column',
+      attrs: {
+        placeholder: 'Select a column',
+      },
+    },
+    evolutionType: {
+      type: 'string',
+      title: 'Compute evolution versus',
+      enum: ['vsLastYear', 'vsLastMonth', 'vsLastWeek', 'vsLastDay'],
+    },
+    uniqueIndexCols: {
+      type: 'array',
+      minItems: 0,
+      items: {
+        type: 'string',
+        minLength: 1,
+      },
+      title: 'uniqueIndexCols',
+      description: 'Index columns',
+      attrs: {
+        placeholder: 'Select columns',
+      },
+    },
+    absEvolColName: {
+      type: 'string',
+      minLength: 1,
+      title: 'Column name for the evolution in absolute value',
+      description: 'Column name for the evolution in absolute value',
+    },
+    pctEvolColName: {
```

Comment:
I think it is weird to produce 2 output columns, when maybe only one is useful. What if we output only one column. It can be absolute value or percentage value depending on a other parameter like: "compute_in_percentage: true / false" (for instance).